src/lib/RpcApi.ts

```diff
@@ -267,7 +267,9 @@ export default class RpcApi {
                 }
 
                 if (error.message === 'Request aborted') {
-                    if (window.opener) {
+                    const rpcState = RpcState.fromJSON(state.rpcState);
```

Comment:
The rpcState is also available as `this._staticStore.rpcState`, because the state was already recovered in line 259.

Reply:
You are right. Fixed it.